tests/src/Context/MessageContext.php

```diff
@@ -16,6 +16,7 @@
  * @package Drupal\nexteuropa\Context
```

Comment:
I will change the permissions. For the git configuration, not sure because what if we have to change permissions on a file on purpose?